### PR TITLE
STIG Hardening: minor fixes

### PR DESCRIPTION
**Pull request by taroth21**
### PR creator: Description

Fixes the validity issue for openSUSE  by profiling the complete section for SLES/SLED.
Also fixed some typos and tweaked some entities. As OpenSCAP was mentioned for fixing
any remaining issues manually, added a link to the OpenSCAP paper, too.  

### PR creator: Are there any relevant issues/feature requests?
* jsc#SLE-25039

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/deployment_yast_installer.xml [hunk 1]
```diff
@@ -2447,8 +2447,8 @@ sle-live-patching 8c541494</screen>
    </important>
    <para>
     This category allows hardening your system with &openscap; security
-    policies. The first policy that was implemented is the <literal>&disa;
-     &stiga;</literal>; more policies will be added in future releases.
+    policies. The first policy that was implemented is the
+   <literal>&stig; (&stiga;)</literal> by the <orgname>&disa;</orgname>.
```
Comment (cwickert) [inline xml/deployment_yast_installer.xml:2451]: The reason why I use `&disa; &stiga`; is that resolves to the string in the installer. While I like your change I think we also need introduce the abbreviation `&disaa;`  to use it later (in line 2468).

```suggestion
   <literal>&stig; (&stiga;)</literal> by the <orgname>&disa;</orgname>
   (<orgname>&disaa;</orgname>).
```

File: xml/deployment_yast_installer.xml [hunk 2]
```diff
@@ -2460,24 +2460,28 @@ sle-live-patching 8c541494</screen>
...
    </tip>
    <para>
     The installer does not check all rules of the profile but only those
-    necesary for the installation or that are hard to fix afterward. To apply
+    necessary for the installation or that are hard to fix afterward. To apply
     the remaining rules, a full SCAP remediation is performed on first boot. You
     can also perform a <guimenu>scan only</guimenu> or <guimenu>do
      nothing</guimenu> and manually remediate the system later with &openscap;.
-    For more information, refer to the article
+    For more information, refer to the articles
```
Comment (cwickert) [inline xml/deployment_yast_installer.xml:2479]: Why plural? It is only one article, isn't it?

Comment (taroth21) [reply xml/deployment_yast_installer.xml:2480]: No, it's two articles, one about OpenSCAP and one about STIG.

**Review by cwickert — APPROVED** [review]
OK then.

File: xml/deployment_yast_installer.xml (hunk 1 above)
Comment (taroth21) [reply xml/deployment_yast_installer.xml]: Good point (did not look at the UI string), makes sense!

**Pull request merged**